install_config/persistent_storage/topics/glusterfs_example_basic.adoc

```diff
@@ -0,0 +1,50 @@
+. In your inventory file, add `glusterfs` in the `[OSEv3:children]` section to
+enable the `[glusterfs]` group:
++
+----
+[OSEv3:children]
+masters
+nodes
+glusterfs
+----
+
+. (Optional) Include any of the following role variables in the `[OSEv3:vars]`
+section you wish to change:
++
+----
+[OSEv3:vars]
+openshift_storage_glusterfs_namespace=glusterfs <1>
+openshift_storage_glusterfs_name=storage <2>
+----
+<1> The project (namespace) to host the storage pods. Defaults to `glusterfs`.
+<2> A name to identify the GlusterFS cluster, which will be used in resource
+names. Defaults to `storage`.
+
+. Add a `[glusterfs]` section with entries for each storage node that will host
+the GlusterFS storage. For each node, set `glusterfs_devices` to a list of raw
+block devices that will be completely managed as part of a GlusterFS cluster.
+There must be at least one device listed. Each device must be bare, with no
+partitions or LVM PVs. Specifying the variable takes the form:
++
+----
+<hostname_or_ip> glusterfs_devices='[ "</path/to/device1/>", "</path/to/device2>", ... ]'
+----
++
+For example:
++
+----
+[glusterfs]
+node11.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node12.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node13.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+----
+
+. Add the hosts listed under `[glusterfs]` to the `[nodes]` group:
++
+----
+[nodes]
+...
+node11.example.com openshift_schedulable=True
```

Comment:
Should this be added to the end of each of the 3 lines - openshift_node_labels="{'role': 'app'}"

Reply:
I pointedly did not add the node selectors to the smaller examples as they don't add anything meaningful. Having both GlusterFS groups in the full example warrants the labels to distinguish why you would want to have both.